scipy/stats/_qmc.py

```diff
@@ -1174,15 +1191,26 @@ class LatinHypercube(QMCEngine):
 
     def __init__(
         self, d: IntNumber, *, centered: bool = False,
+        scramble: bool = True,
         strength: int = 1,
         optimization: Optional[Literal["random-cd", "lloyd"]] = None,
         seed: SeedType = None
     ) -> None:
+        if centered:
+            scramble = False
+            warnings.warn(
+                "'centered' is deprecated and will be removed in SciPy 1.11."
+                " Please use 'scramble' instead. 'centered=True' corresponds"
+                " to 'scramble=False'. Hence the value of 'scramble' was"
+                " changed to False.",
```

Comment:
> Hence the value of 'scramble' was changed to False.",

That's an implementation detail. If `centered` is still documented, I don't think you need to mention that you now implement `centered` by setting `scramble`.